### ISSUE TITLE
Unexpected token `[` in foreach.

### ISSUE THREAD

**Issue by Gixx** [phpmd/phpmd#592]
PHP 7.1
PHPMD 2.6.0

Code:
```
<?php

$arr = [
    ['country' => 'Germany', 'code' => 'DE'],
    ['country' => 'France', 'code' => 'FR'],
    ['country' => 'U.S.A.', 'code' => 'US'],
];

foreach($arr as ['country' => $countryName, 'code' => $countryCode]) {
    echo $countryName.' is '.$countryCode.PHP_EOL;
}
```

Output:
> Unexpected token: [, line: 9, col: 17

**Comment by baer95** [comment]
Just had the same problem. PHP 7.2, PHPMD 2.6.0.

**Comment by baer95** [comment]
related issue: https://github.com/phpmd/phpmd/issues/589

**Comment by alanpoulain** [comment]
This error will be solved when https://github.com/pdepend/pdepend/pull/383/files will be merged in pdepend.

**Comment by kylekatarnls** [comment]
This is fixed since PDepend 2.6.0.
